content/blog/architect-aws-application-infra-with-pulumi-stack-references/index.md

```diff
@@ -0,0 +1,457 @@
+---
+title: "Architect AWS Application Infrastructure with Pulumi Stack References"
+date: "2019-06-14"
+meta_desc: "How to architect your AWS infrastructure to optimize team collaboration with Pulumi Stack References"
+meta_image: "dashboard.png"
+authors: ["paul-stack"]
+tags: ["AWS","stack-reference", "architecture"]
+---
+
+In this post, we will talk about the best way to architect your Pulumi applications. We are going to build out the following
+infrastructure in AWS:
+
+* AWS Fargate service that does not serve traffic directly
+* AWS ALB as the entry point to the Fargate Service
+* AWS RDS Instance that is stored in a separate network from the Application and does not service traffic directly from the internet
```

Comment:
s/network/VPC, s/does not service traffic directly from the Internet/is only accessible from the application VPC/?